stories/user-menu/user-menu.stories.js

```diff
@@ -0,0 +1,981 @@
+                                        <a tabindex="0" class="fd-list__link">
+                                            <i role="presentation" class="fd-list__icon sap-icon--time-account"></i>
+                                            <span class="fd-list__title">Recent Activity</span>
+                                        </a>
+                                    </li>
+                                    <li tabindex="-1" role="listitem" class="fd-list__item fd-list__item--link">
+                                        <a tabindex="0" class="fd-list__link">
+                                            <i role="presentation" class="fd-list__icon sap-icon--activity-individual"></i>
+                                            <span class="fd-list__title">Frequently Used</span>
+                                        </a>
+                                    </li>
+                                    <li tabindex="-1" role="listitem" class="fd-list__item fd-list__item--link">
+                                        <a tabindex="0" class="fd-list__link">
+                                            <i role="presentation" class="fd-list__icon sap-icon--detail-view"></i>
+                                            <span class="fd-list__title">Content Finder</span>
+                                        </a>
+                                    </li>
+                                    <li tabindex="-1" role="listitem" class="fd-list__item fd-list__item--link">
+                                        <a tabindex="0" class="fd-list__link fd-list__link--navigation-indicator is-selected">
+                                            <i role="presentation" class="fd-list__icon sap-icon--employee-lookup"></i>
+                                            <span class="fd-list__title">Proxy User</span>
+                                        </a>
+                                    </li>
+                                    <li tabindex="-1" role="listitem" class="fd-list__item fd-list__item--link">
+                                        <a tabindex="0" class="fd-list__link fd-list__link--navigation-indicator">
+                                            <i role="presentation" class="fd-list__icon sap-icon--role"></i>
+                                            <span class="fd-list__title">Proxy Role</span>
+                                        </a>
+                                    </li>
+                                    <li tabindex="-1" role="listitem" class="fd-list__item fd-list__item--link">
+                                        <a tabindex="0" class="fd-list__link fd-list__link--navigation-indicator">
+                                            <i role="presentation" class="fd-list__icon sap-icon--history"></i>
+                                            <span class="fd-list__title">Delegate</span>
+                
+                                            <span class="fd-list__secondary">
```

Comment:
I think it's better to use `fd-list__item-counter` instead of `fd-list__secondary`. `fd-list__secondary` class sets the line-height to 1rem, which is related to truncation and we can't get rid of it. This is "cutting" the bottom of the badge and it looks broken:
<img width="875" alt="Screen Shot 2021-04-16 at 11 52 47 AM" src="https://user-images.githubusercontent.com/39598672/115050983-51446d80-9eaa-11eb-9e97-83dd6874a0ff.png">

By applying `fd-list__item-counter` you get:
<img width="799" alt="Screen Shot 2021-04-16 at 11 52 21 AM" src="https://user-images.githubusercontent.com/39598672/115051033-5d302f80-9eaa-11eb-9b48-d4e9f15eb428.png">